frontend/app/controllers/spree/orders_controller.rb

```diff
@@ -49,6 +49,10 @@ def populate
       if quantity.between?(1, 2_147_483_647)
         begin
           order.contents.add(variant, quantity, options)
+          order.update_line_item_prices!
+          order.create_tax_charge!
+          order.update_totals
+          order.persist_totals
```

Comment:
I think it would be better to user `order.update_with_updater!` instead of these last two lines.